internal/runtime/metrics/metrics.go

```diff
@@ -46,8 +46,8 @@ var (
 		prometheus.NewCounterVec(prometheus.CounterOpts{
 			Subsystem: runtimeSDKSubsystem,
 			Name:      "requests_total",
-			Help:      "Number of HTTP requests, partitioned by status code, host and hook.",
-		}, []string{"code", "host", "group", "version", "hook"}),
+			Help:      "Number of HTTP requests, partitioned by code, status, host and hook.",
```

Comment:
```suggestion
			Help:      "Number of HTTP requests, partitioned by status code, host, hook and response status.",
```

Reply:
Let's keep it in the order of the code here :-)

Reply:
done, thanks!